### ISSUE TITLE
Workshop/Session/ColaboFlow Management/Design

### ISSUE THREAD

**Issue by sinisarudan** [Cha-OS/colabo#432]
- defining **participants number**
  - thus we can always show/know how many and which users haven't done something;
    - how many haven't logged in
  - some of them might be only supported through the system and not having their Access Point (mobile phone/computer) for interaction
- this issue is related to the #334
- using ColaboFlow to define (visually) the workshop procedure - steps user go through the workshop
  - maybe it automatically drives them through these steps/pages in the app

**Comment by mprinc** [comment]
Yes, drawing them, and showing their state on the moderator page, or collective page is what we want
